baseplate/lib/experiments/__init__.py

```diff
@@ -44,9 +44,13 @@ class ExperimentsContextFactory(ContextFactory):
     """
 
     def __init__(
-        self, path: str, event_logger: Optional[EventLogger] = None, timeout: Optional[float] = None
+        self,
+        path: str,
+        event_logger: Optional[EventLogger] = None,
+        timeout: Optional[float] = None,
+        backoff: Optional[float] = None,
```

Comment:
can you add documentation of this new parameter to the docstring above please?

Reply:
Done.